sql/src/main/java/io/crate/operation/udf/UserDefinedFunctionDDLDispatcher.java

```diff
@@ -36,17 +38,27 @@
 @Singleton
 public class UserDefinedFunctionDDLDispatcher {
 
+    private final Settings settings;
```

Comment:
Use local variable isEnabled to store it and avoid checking the settings every time